server/journeys/referral/on-behalf-of/view.njk

```diff
@@ -1,14 +1,48 @@
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% set pageTitle = "Make a CSIP referral - Digital Prison Services" %}
 {% extends "../../../views/pages/index.njk" %}
 
 {% block content %}
+<div class="govuk-grid-column-two-thirds">
+  <h1 class="govuk-heading-l">Make a CSIP referral</h1>
+
+  <p>You can use this service to refer {{ prisoner.firstName }} {{ prisoner.lastName }} for a Challenge, Support and Intervention Plan (CSIP)</p>
```

Comment:
Should use `{{ (prisoner.firstName + ' ' + prisoner.lastName) | convertToTitleCase }}` to correctly case prisoner name. Note you'll need to add it to nunjucksSetup.ts.

Also has a full stop after (CSIP) in the design

Reply:
Fixed